deps.edn

```diff
@@ -0,0 +1,24 @@
+{:aliases
+ {:1.7  {:extra-deps #:org.clojure {clojure #:mvn {:version "1.7.0"}}},
+  :1.8  {:extra-deps #:org.clojure {clojure #:mvn {:version "1.8.0"}}},
+  :1.9  {:extra-deps #:org.clojure {clojure #:mvn {:version "1.9.0"}}},
+  :1.10 {:extra-deps #:org.clojure {clojure #:mvn {:version "1.20.0"}}}
+  :runner
+  {:extra-deps
+   #:com.cognitect {test-runner
+                    {:git/url
+                     "https://github.com/cognitect-labs/test-runner",
+                     :sha "76568540e7f40268ad2b646110f237a60295fa3c"}},
+   :main-opts      ["-m" "cognitect.test-runner" "-d" "test"]},
+  :test {:extra-deps  {org.clojure/test.check #:mvn {:version "RELEASE"}}
+         :extra-paths ["test"]}
+  :dev  {:jvm-opts ["-Dclojure.spec.skip-macros=true"
```

Comment:
Why are we skipping spec checks of the macros?

Reply:
fixed